packages/core/src/types.ts

```diff
@@ -19,12 +20,13 @@ export interface UserOperationCallData {
 
 export type BatchUserOperationCallData = UserOperationCallData[];
 
-export type UserOperationOverrides = Partial<
-  Pick<
-    UserOperationStruct,
-    "maxFeePerGas" | "maxPriorityFeePerGas" | "paymasterAndData"
-  >
->;
+export type UserOperationOverrides = Partial<{
```

Comment:
can we avoid dupes with `UserOperationStruct` by making this an `Omit` or `Pick` on `UserOperationStruct` again?